src/main/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSpark.java

```diff
@@ -231,7 +253,8 @@ public static void callVariantsWithHaplotypeCallerAndWriteOutput(
         final Broadcast<ReferenceMultiSource> referenceBroadcast = ctx.broadcast(reference);
         final Broadcast<HaplotypeCallerArgumentCollection> hcArgsBroadcast = ctx.broadcast(hcArgs);
 
-        final VariantAnnotatorEngine variantAnnotatorEngine = VariantAnnotatorEngine.ofSelectedMinusExcluded(hcArgs.defaultGATKVariantAnnotationArgumentCollection, hcArgs.dbsnp.dbsnp, hcArgs.comps, hcArgs.emitReferenceConfidence == ReferenceConfidenceMode.GVCF);
+        //TODO getAnnotationsToUse() cant be referenced from static context but its what we need to invoke here
+        final VariantAnnotatorEngine variantAnnotatorEngine = new VariantAnnotatorEngine(Collections.emptyList(), hcArgs.dbsnp.dbsnp, hcArgs.comps, hcArgs.emitReferenceConfidence == ReferenceConfidenceMode.GVCF);
```

Comment:
This will have to be fixed somehow. Will probably require a little bit of refactoring.

Reply:
I had already threaded it through, it seems i had forgotten that we actually construct the HaplotypeCallerEngine twice, one of which we only use for header construction. Thus, I missed the second (and more important one) when threading the variant annotator engine through.